lib/ap/crt/crt1.S

```diff
@@ -6,10 +6,26 @@
 .globl	_start
 _start:
 
+	// 24 bytes room for environ
```

Comment:
this is a confusing comment .The environ needs 24 bytes? Or ... it makes no sense to me.